README.md

```diff
@@ -11,3 +11,12 @@ _**Prerequisits: This repository requires node v16 and up**_
 1. Clone this repository
 2. Install dependecies with `npm i`
 3. Test and lint with `npm test`
+
+## Manual Deployment
+
+Before making a deploy make sure you make a build locally of the content that should be deployed by running `npm run buildPreview`.
+
+Afterwards, run `npm run netlify deploy` to run manual deployments to a unique address that will be generated at the time of deploy. 
+To deploy to [https://new-lexeme-special-page.netlify.app](https://new-lexeme-special-page.netlify.app) run `npm run netlify deploy --prod`. 
+
+To be able to run the netlify commands you will need to run `netlify login` once after installing netlify to authenticate and obtain a token.
```

Comment:
What does installing netlify mean here? Is it part of the `npm install` or something else? (And if it is part of the `npm install`, shouldn’t the command be `npm run netlify login`, so it uses the `netlify` command from `node_modules`?)

Reply:
ah yes, it should!!! my bad. i will fix it. installing netlify means running `npm install netlify-cli --save-dev`

Reply:
Done, the command is updated!